lib/phoenix_starter/users/user_notifier.ex

```diff
@@ -2,68 +2,65 @@ defmodule PhoenixStarter.Users.UserNotifier do
   @moduledoc """
   Delivers notifications to `PhoenixStarter.Users.User`s.
   """
+  use Phoenix.Swoosh, view: PhoenixStarterWeb.UserEmailView
+
+  import PhoenixStarterWeb.Gettext
+
   alias PhoenixStarter.Users.User
-  alias PhoenixStarter.Users.UserEmail
   alias PhoenixStarter.Mailer
-  alias PhoenixStarter.Workers.UserEmailWorker
-
-  @type notifier_result ::
-          {:ok, Swoosh.Email}
-          | {:ok, Oban.Job.t()}
-          | {:error, Oban.Job.changeset()}
-          | {:error, term()}
+  alias PhoenixStarter.Email
 
   @doc """
   Deliver instructions to confirm `PhoenixStarter.Users.User`.
   """
-  @spec deliver_confirmation_instructions(User.t(), String.t(), boolean()) :: notifier_result
-  def deliver_confirmation_instructions(user, url, async \\ true)
-
-  def deliver_confirmation_instructions(user, url, true) do
-    %{email: "confirmation_instructions", user_id: user.id, url: url}
-    |> UserEmailWorker.new()
-    |> Oban.insert()
```

Comment:
The Swoosh README (written by Jose) recommends not doing this unless you have a business case for it, so by default I'm removing this code. It can be added later if the specific app requires it.